### PR TITLE
chore(ui): enable eslint rules for i18next

### PR DESCRIPTION
**Pull request by chirag-madlani**
add sync language resources with main file

### Describe your changes :

I worked on the ..... because ...

- Improving developer experience with 18next 
- Added script to sync localisation resource file with `en-us.json` file

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 1]
```diff
@@ -22,6 +23,7 @@
     "added-lowercase": "added",
     "added-yet-lowercase": "added yet.",
     "adding-new-classification": "Adding New Classification",
+    "adding-new-entity": "Adding a new {{entity}} is easy, just give it a spin!",
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:26]: For label, Key must represent actual text

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 2]
```diff
@@ -103,11 +123,15 @@
     "count": "Count",
     "create": "Create",
     "create-entity": "Create {{entity}}",
+    "create-new-test-suite": "Create new test suite",
     "created-a-task-lowercase": "created a task",
     "created-by-me": "Created by me",
     "created-lowercase": "created",
+    "creating-account": "Creating Account ....",
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:130]: Let's remove three dots from translation

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 3]
```diff
@@ -455,12 +545,15 @@
...
     "secret-key": "Secret Key",
+    "select": "Select",
     "select-a-chart": "Select a Chart",
     "select-a-metric-type": "Select a Metric Type",
     "select-a-policy": "Select a Policy",
     "select-column-plural-to-exclude": "Select Columns to Exclude",
     "select-column-plural-to-include": "Select Columns to Include",
     "select-field": "Select {{field}}",
+    "select-teams": "Select teams",
+    "select-type": "Select type",
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:556]: Can we use `select-field` here?

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 4]
```diff
@@ -514,20 +612,26 @@
...
+    "test-connection": "Test Connection",
     "test-entity": "Test {{entity}}",
     "test-plural": "Tests",
     "test-suite": "Test Suite",
     "test-suite-status": "Test Suite Status",
+    "testing-connection": "Testing Connection",
     "tests-summary": "Tests Summary",
     "thread-count": "Thread Count",
+    "three-dash-symbol": "---",
+    "three-dots-symbol": "•••",
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:624]: I am not confident to have symbols on translation any specific reason for adding them

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 5]
```diff
@@ -564,13 +670,17 @@
     "username": "Username",
     "username-or-email": "Username or Email",
     "users": "Users",
+    "valid-condition": "✅ Valid condition",
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:673]: Let's remove check emoji from text

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 6]
```diff
@@ -611,13 +725,16 @@
     "assigned-you-a-new-task-lowercase": "a new task has been assigned to you",
     "at-least-one-policy": "Enter at least one policy",
     "bot-email-confirmation": "{{email}} for {{botName}} bot",
+    "closed-this-task": "closed this task",
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:728]: It should be label

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 7]
```diff
@@ -785,12 +936,16 @@
...
     "view-deleted-teams": "View All the Deleted Teams, which come under this Team.",
+    "view-sample-data": "To view Sample Data, run the Profiler Ingestion. Please refer to this doc to schedule the",
     "view-sample-data-message": "To view Sample Data, run the MetaData Ingestion. Please refer to this doc to schedule the",
+    "viewing-older-version": "Viewing older version \n Go to latest to update details",
     "webhook-listing-message": "The webhook allows external services to be notified of the metadata change events happening in your organization through APIs. Register callback URLs with webhook integration to receive metadata event notifications. You can add, list, update, and delete webhooks.",
     "webhook-type-listing-message": "Provide timely updates to the producers and consumers of metadata via {{webhookType}} notifications. Use {{webhookType}} webhooks to send notifications regarding the metadata change events in your organization through APIs. You can add, list, update, and delete these webhooks.",
+    "welcome-to-om": "Welcome to OpenMetadata!",
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:948]: I feel we have same in label too can you check?

File: openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/AddDataQualityTestV1.tsx [hunk 8]
```diff
@@ -13,8 +13,9 @@
 
 import { Col, Row, Typography } from 'antd';
 import { AxiosError } from 'axios';
+import { t } from 'i18next';
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/AddDataQualityTestV1.tsx:16]: let's use useTranslate hook for component

**Review by ShaileshParmar11 — APPROVED** [review]
Many places we have used {t} function directly instead of importing the hook and using it!

**Pull request merged**